examples/llm_compression/openvino/tiny_llama/main.py

```diff
@@ -67,7 +67,7 @@ def transform_fn(data, model, tokenizer):
     )
     model.save_pretrained(OUTPUT_DIR)
 
-    model = OVModelForCausalLM.from_pretrained(OUTPUT_DIR)
```

Comment:
Do we also need to set "KV_CACHE_PRECISION": "u8"`?

Reply:
in my experiments it doesn't change anything